src/utils.rs

```diff
@@ -48,9 +52,20 @@ pub fn get_content_type(headers: &HeaderMap) -> Option<ContentType> {
 }
 
 // https://github.com/seanmonstar/reqwest/issues/646#issuecomment-616985015
-pub async fn body_to_file(path: impl AsRef<Path>) -> reqwest::Body {
-    let file = File::open(&path).await.unwrap();
-    reqwest::Body::wrap_stream(FramedRead::new(file, BytesCodec::new()))
+pub async fn file_to_part(path: impl AsRef<Path>) -> io::Result<multipart::Part> {
+    let path = path.as_ref();
+    let file_name = path
+        .file_name()
+        .map(OsStr::to_string_lossy)
+        .unwrap_or_else(|| "file".into())
```

Comment:
This default value could lead to some seemingly-strange behavior from the perspective of a user.
"Why is it looking for a file named 'file'? I didn't tell it to do that."

Reply:
I figured a default value would be better than crashing, but I realize now that it's possible to leave out the value altogether. Thank you.